docs/release_notes/v1.7.0.adoc

```diff
@@ -0,0 +1,48 @@
+[id="{p}-release-notes-v1.7.0"]
+=== Elastic Cloud Control (ecctl) v1.7.0
+++++
+<titleabbrev>v1.7.0</titleabbrev>
+++++
+
+<<{p}-release-notes-v1.7.0-whats-new,What's new>> | <<{p}-release-notes-v1.7.0-bug-fixes,Bug fixes>> | <<{p}-release-notes-v1.7.0-breaking-changes,Breaking changes>> | <<{p}-release-notes-v1.7.0-changelog,Changelog>>
```

Comment:
These references aren't pointing anywhere now now that we've removed the headings, we should ammend the links otherwise the docs build might break